charts/kargo/values.yaml

```diff
@@ -449,6 +449,8 @@ controller:
     warehouses:
       ## @param controller.reconcilers.warehouses.maxConcurrentReconciles optionally overrides the maximum number of Warehouse resources the controller can reconcile concurrently.
       maxConcurrentReconciles:
+      ## @param controller.reconcilers.warehouses.minReconciliationInterval optionally sets the minimum reconciliation interval for Warehouse resources. Accepts duration format (e.g., "5m", "1h", "30s"). If a Warehouse specifies an interval lower than this minimum, the minimum value will be enforced instead. If not set, no minimum is enforced.
```

Comment:
Enforcing no minimum if not set seems as if it may be asking for trouble. It's easy mor me to imagine an operator overlooking this setting and many developers, impatient with the amount of time their Warehouses require to detect new artifacts, doing the exact wrong thing by turning the interval down quite low on every Warehouse.

imho, we should default this to 5m.